asciidoctor-stylesheet/sass/devonfw.scss

```diff
@@ -222,17 +222,97 @@ tr:nth-child(even) {
   }
 }
 
+
+
+#content {
+  max-width: 90em;
+  padding-left: 1em;
+  padding-right: 1em;
+
+}
+
 @media screen and (min-width: 768px) {
   .sect1 {
     padding-bottom: 3rem;
   }
+  body.book.toc2.toc-left {
+    display: grid;
+    grid-template-rows: auto 0 40px 1fr auto;
+    grid-template-columns: calc(90em * 0.01) calc(90em * 0.20) calc(90em * 0.52) calc(90em * 0.01) calc(90em * 0.25);
+    padding-left: 0;
 }
 
-#content {
-  max-width: 90em;
-  padding-left: 1em;
-  padding-right: 1em;
+    body.book.toc2.toc-left .website-navbar {
+    max-width: 100vw;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    grid-column-start: 1;
+    grid-column-end: 7;
+}
+
+    body.book.toc2.toc-left #header > :nth-child(3) {
+    grid-row-start: 3;
+    position: relative;
+    background-color: white;
+    border: none;
+    grid-column-start: 1;
+    grid-column-end: 2;
+    width: 66%;
+    padding-right: 2rem;
+}
+
```

Comment:
Actually, you solve layout with bootstrap because otherwise it becomes very complex when you tinker. Also, the current design is a bit broken:
- Data (updated at) is top right and not bottom as before.
- at a width of e.g. 800 page is completely shot (nothing to see)
- Footer is somehow left aligned and not centered (cf. current website).
   Solution:
   ```sectlevel3 {
    height: 100%; <- remove, not needed
    
     body.book.toc2.toc-left #footer {
        max-width: 100vw;
        margin-left: 0;
        grid-row-start: 5;
        grid-column-start: 1;
        grid-column-end: 7;
       }
      (andere Stelle im Code)